PhoneBook.cs

```diff
@@ -0,0 +1,113 @@
+﻿using PhoneBook.StaticHelpers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace PhoneBook
+{
+    public class PhoneBook
+    {
+        private const int MembersCount = 4;
+        private const int MembersCountWithoutSurname = 3;
```

Comment:
Put in constants. Also not understandable what Members Count this variable represents